### PR TITLE
APIs functionality

### PR DESCRIPTION
**Pull request by kennyegun24**
## In this PR, I implemented the following: 
- Add books to the API 
- Refresh button to display books when clicked on

### CODE REVIEW

File: src/modules/api.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import Scores from './scores.js';
+
+const display = (gameData) => {
+  const scoreDisplay = document.querySelector('.scores_name');
+  scoreDisplay.innerHTML = '';
+  // gameData.sort((a, b) => a.score - b.user);
```
Comment (aimalamiri) [inline src/modules/api.js]: To keep your codebase cleaner it's recommended not to leave commented code. So please make sure to remove it.

File: src/modules/api.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  const displayScores = gameData.map((list) => `<p class ='scoreList'>${list.user}: ${list.score} </p>`).join('');
+  scoreDisplay.innerHTML = displayScores;
+};
+
+const addScore = async (user, score) => {
+  const apiURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+  const url = `${apiURL}games/`;
+  const ID = 'ajtuTgubydhYUNubdauu';
```
Comment (aimalamiri) [inline src/modules/api.js]: This part of the code is repeated multiple times, you can export this to a new function or variable to make it reusable. So it's highly recommended to follow the **[DRY](https://codinglead.co/javascript/what-is-DRY-code)** principle.

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @kennyegun24, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

-  The API is used properly ✔️ ⭐
- Good job for using the right GitFlow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


- [ ] [**Optional**] Everything works very well, but when I submit the form it is sending the score to the API and also fetching it. But the table is not updated, It's highly recommended to update the table once you fetch the data after the form submission.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/api.js (hunk 2 above)
Comment (kennyegun24) [reply src/modules/api.js]: Thanks, I have made correction to this

File: src/modules/api.js (hunk 1 above)
Comment (kennyegun24) [reply src/modules/api.js]: THanks, I have made correction to this and the optional suggestioon too

**Review by prolajumokeoni — APPROVED** [review]
Hi @kennyegun24 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**